SimG4CMS/Calo/interface/HCalSD.h

```diff
@@ -97,11 +99,13 @@ class HCalSD : public CaloSD, public Observer<const BeginOfJob *> {
   bool                          testNumber, neutralDensity, testNS_;
   double                        birk1, birk2, birk3, betaThr;
   bool                          useHF, useShowerLibrary, useParam, applyFidCut;
+  bool                          isPametrized;
```

Comment:
can we spell this correctly as `isParametrized`?

Reply:
@kpedro88, Sure. 
Note, that I do not yet include improvements you proposed, because first of all need to see comparison results - in the previous attempt there were too many differences, likely due to change of sim histories.